playlists/excel.yaml

```diff
@@ -14,7 +13,6 @@
   fileName: basic-api-call-es5.yaml
   relativePath: excel\01-basics\basic-api-call-es5.yaml
```

Comment:
Is this "relativePath" property used anywhere (e.g., in Script Lab?).  I don't think so.  If so, let's remove it, and cut down on the bandwidth...